apps/prairielearn/src/server.js

```diff
@@ -78,13 +78,6 @@ process.on('warning', (e) => console.warn(e));
 
 const argv = yargsParser(process.argv.slice(2));
 
-// If there is only one argument and `server.js` is being executed directly,
-// legacy it into the config option.
-if (require.main === module && argv['_'].length === 1) {
-  argv['config'] = argv['_'][0];
-  argv['_'] = [];
-}
-
```

Comment:
Hopefully it's safe to break this legacy feature now? We certainly don't rely on it for production deployments, and it's been over 6 years since we introduced `--config`.

Reply:
Yeah, I think it's fine to break this. Just to make sure I understand, if someone is in fact using the old syntax then it will break with an error message, right? It won't just silently break?

Reply:
No, it will in fact silently break; `yargs-parser` doesn't do any validation.